syntax = "proto3";

package grpc;

service Updater {
  rpc  sendModel(updateRequest) returns (updateReply) {}
}

message updateRequest {
  bytes model = 1;
  uint64 whole_size = 2;
  uint64 batch_size = 3;
}

message updateReply {
  bytes model = 1;
}